packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -236,6 +242,36 @@ _Props/Attrs_
 -   `markdown` - for retrieving and modifying the markdown value. If the client modifies the markdown value, it will be parsed into a Node using the
     [prosemirror-markdown parser](https://github.com/ProseMirror/prosemirror-markdown/blob/9049cd1ec20540d70352f8a3e8736fb0d1f9ce1b/src/from_markdown.ts#L199).
     The parsed node will then be rendered in the viewer component as rich text.
+-   `link-target` - is an attribute that accepts valid `target` values for the `anchor` element, applicable to all links within the viewer component.
+    The following values are accepted by this attribute:
+    1.  `_self`: the current browsing context. (Default)
+    2.  `_blank`: usually a new tab, but users can configure browsers to open a new window instead.
+    3.  `_parent`: the parent browsing context of the current one. If no parent, behaves as `_self`.
+    4.  `_top`: the topmost browsing context (the "highest" context that's an ancestor of the current one). If no ancestors, behaves as `_self`.
+
+Above definitions are from the [MDN reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#:~:text=separated%20link%20types.-,target,-Where%20to%20display).
```

Comment:
The extra context is not needed, can just say it aligns with the anchor target attribute exactly and link to the mdn reference. Also to be clear the target attribute is not an enum, it takes arbitrary strings. We should behave exactly like the built-in target attribute.

Reply:
Updated the spec.